README.md

```diff
@@ -0,0 +1,104 @@
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+[![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
+
+# vLLM Backend
+
+The Triton backend for [vLLM](https://github.com/vllm-project/vllm).
+You can learn more about Triton backends in the [backend
+repo](https://github.com/triton-inference-server/backend). Ask
+questions or report problems on the [issues
+page](https://github.com/triton-inference-server/server/issues).
+This backend is designed to run [vLLM](https://github.com/vllm-project/vllm)
+with
+[vLLM supported models](https://vllm.readthedocs.io/en/latest/models/supported_models.html)
+it supports.
+
+Where can I ask general questions about Triton and Triton backends?
+Be sure to read all the information below as well as the [general
+Triton documentation](https://github.com/triton-inference-server/server#triton-inference-server)
+available in the main [server](https://github.com/triton-inference-server/server)
+repo. If you don't find your answer there you can ask questions on the
+main Triton [issues page](https://github.com/triton-inference-server/server/issues).
+
+## Build the vLLM Backend
+
+As a Python-based backend, your Triton server just needs to have the [Python backend](https://github.com/triton-inference-server/python_backend)
+located in the backends directory: `/opt/tritonserver/backends/python`. After that, you can save the vLLM backend in the backends folder as `/opt/tritonserver/backends/vllm`. The `model.py` file in the `src` directory should be in the vllm folder and will function as your Python-based backend.
+
+In other words, there are no build steps. You only need to copy this to your Triton backends repository. If you use the official Triton vLLM container, this is already set up for you.
```

Comment:
There are a couple of options, how you can build  vLLM backend. 
Option 1. You can follow steps described in [Building With Docker](https://github.com/triton-inference-server/server/blob/main/docs/customization_guide/build.md#building-with-docker) and use `build.py` script.
The sample command will build a Triton Server container with all available options enabled:
 ```
./build.py -v --image=base,${BASE_CONTAINER_IMAGE_NAME}
                 --enable-logging --enable-stats --enable-tracing
                 --enable-metrics --enable-gpu-metrics --enable-cpu-metrics
                 --enable-gpu
                 --filesystem=gcs --filesystem=s3 --filesystem=azure_storage
                 --endpoint=http --endpoint=grpc --endpoint=sagemaker --endpoint=vertex-ai
                 --backend=python:r23.10
                 --backend=vllm:r23.10
```
Option 2. You can install vLLM backend directly into our NGC Triton container. In this case, please install vllm first: `pip install vllm`, then set up vllm_backend in the container as follows:
```
mkdir -p /opt/tritonserver/backends/vllm_backend
wget -P /opt/tritonserver/backends/vllm_backend https://raw.githubusercontent.com/triton-inference-server/vllm_backend/main/src/model.py
```


Note: we should also mention separate container at some point

Reply:
Thanks for drafting these instructions. Added!